projects/elements/src/lib/lazy-elements/lazy-elements-loader.service.ts

```diff
@@ -182,6 +206,18 @@ export class LazyElementsLoaderService {
       return Promise.reject(err);
     }
   }
+
+  private resolveImportMap(url: string) {
+    const System = (window as any).System;
+    if (System) {
+      url = System.resolve(url);
+    } else {
+      throw new Error(
+        `${LOG_PREFIX} - SystemJS is not loaded and thus '${url}' can't be resolved, *axLazyElement depends on SystemJS for import map feature`
```

Comment:
Would mention that this error is a result of using `importMap` AND not having `SystemJs`